graphics/vtk/Portfile

```diff
@@ -132,9 +132,11 @@ foreach pyver ${python_versions} {
 
             configure.args-append \
                 -DVTK_WRAP_PYTHON:BOOL=ON \
+                -DVTK_PYTHON_VERSION:STRING=3 \
```

Comment:
I'm pretty sure this is not needed.

Reply:
OK, I removed it. I have not retested, but I think you are right that it is not used.